include/pmacc/particles/memory/boxes/ExchangePushDataBox.hpp

```diff
@@ -42,12 +42,12 @@ namespace pmacc
 
         HDINLINE ExchangePushDataBox(
             VALUE* data,
-            TYPE* currentSizePointer,
+            TYPE* ptrToSize,
```

Comment:
Is it required or a good idea to call pointers to objects as ptrToObj? is it not evident from the type

Reply:
I changed it to `particleCounter`